UnityProject/Assets/Scripts/Items/Food/XenomorphFood.cs

```diff
@@ -52,7 +52,7 @@ public override void TryConsume(GameObject feederGO, GameObject eaterGO)
 			}
 		}
 
-		public override void Eat(PlayerScript eater, PlayerScript feeder)
+		protected override void Eat(PlayerScript eater, PlayerScript feeder)
```

Comment:
Doesn't this mean the contents is duplicated each time you eat?

Reply:
no

Reply:
where does it split the contents of the Reagent mix of the stomach?

Reply:
*of the food

Reply:
Go to the edible component and see for yourself